### ISSUE TITLE
Feature - Support node_modules/.bin

### ISSUE THREAD

**Issue by mcm-ham** [fnando/vscode-linter#31]
Since a number of linters are npm packages it would be great to support node_modules/.bin in addition to $PATH so that we can add the various linters as devDependencies of the current project avoiding the need for developers to install them globally and we can control the version to avoid issues where different developers can be using different versions.

**Comment by oojacoboo** [comment]
Just add `./node_modules/.bin` to your `$PATH`

**Comment by mcm-ham** [comment]
That only works for a single project, not if you're dealing with multiple projects or repositories.

**Comment by oojacoboo** [comment]
@mcm-ham That's not true, it works for any directory where you have a `node_modules` directory located.

**Comment by mcm-ham** [comment]
Ah missed that you were suggesting relative paths, it's a good thought. Unfortunately it didn't work for me, when I try adding  `process.cwd()` to the extension it outputs `/` rather than the project folder which could explain why it doesn't work.
